package.json

```diff
@@ -30,6 +30,7 @@
     "express-sslify": "^1.2.0",
     "helmet": "^3.21.2",
     "hot-shots": "^7.4.1",
+    "ioredis": "^4.16.3",
```

Comment:
bull already requires ioredis, but since we `require` it now in our code, we need it to be in package.json